config/m4/cuda.m4

```diff
@@ -63,6 +63,13 @@ AS_IF([test "x$cuda_checked" != "xyes"],
                [AC_CHECK_LIB([cudart], [cudaGetDeviceCount],
                              [CUDA_LIBS="$CUDA_LIBS -lcudart"], [cuda_happy="no"])])
 
+         AC_CHECK_HEADERS([cooperative_groups.h],
```

Comment:
do we need to check for this? support for cooperative group was added long before CUDA 11 which is min supported in UCC

Reply:
Ok I see, I erased it